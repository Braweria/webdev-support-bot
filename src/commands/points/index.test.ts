import { dbConnect } from '../..';

import pointsHandler from '.';

test('should throw an error if no user has been found', async () => {
  // Connect to database
  dbConnect();

  const msg: any = {
    author: { id: '1', tag: 'test#1234' },
    channel: { send: jest.fn() },
    delete: jest.fn(),
  };

  await pointsHandler(msg);

  expect(msg.channel.send.mock.calls[0]).toMatchSnapshot();
});
