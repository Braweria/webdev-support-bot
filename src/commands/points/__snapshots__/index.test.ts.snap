// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should throw an error if no user has been found 1`] = `
Array [
  Object {
    "embed": Object {
      "author": null,
      "color": 16670549,
      "description": "You have accumulated 0 points.",
      "fields": Array [],
      "footer": Object {
        "iconURL": "https://github.com/ljosberinn/webdev-support-bot/blob/master/logo.png?raw=true",
        "text": "Helpful User Points",
      },
      "title": "test#1234",
      "url": null,
    },
  },
]
`;